src/editors/textEditor/textEditor.js

```diff
@@ -445,6 +445,10 @@ export class TextEditor extends BaseEditor {
   bindEvents() {
     this.eventManager.addEventListener(this.TEXTAREA, 'cut', event => event.stopPropagation());
     this.eventManager.addEventListener(this.TEXTAREA, 'paste', event => event.stopPropagation());
+    if (isIOS()) {
```

Comment:
One suggestion here. I think that it would be better for readability to put a new line before the `if` statement. It's easier to spot different blocks of code while source code analyzing.
```suggestion

    if (isIOS()) {
```

Reply:
@budnix I see this suggestion often in reviews. Maybe an automated rule https://eslint.org/docs/rules/padding-line-between-statements with `{ blankLine: "always", prev: "*", next: "if" }` will help?

Reply:
Definitely worth a try, thanks! I will experiment with that on a separate branch/PR as it affects a lot of files.